bbc

```diff
@@ -10,21 +10,24 @@ def bbc():
 @bbc.command()
 @click.argument('category', required=True)
 @click.option('--limit', type=click.INT, required=False, default=False, help='Limit of entries')
-def news(category, limit):
+@click.option('--edition', type=click.STRING, required=False, default='int', help='edition to use for top stories')
```

Comment:
We should list the possible editions in the help message?

Reply:
Done. This is what the help message looks like:

```
Usage: bbc news [OPTIONS] CATEGORY

Options:
  --limit INTEGER        Limit of entries
  --edition [us|uk|int]  Edition to use for top stories
  --help                 Show this message and exit.
```